third_party/amd/lib/TritonAMDGPUToLLVM/ElementwiseOpToLLVM.cpp

```diff
@@ -1503,12 +1504,13 @@ void populateElementwiseOpToLLVMPatterns(
     int numWarps, ModuleAxisInfoAnalysis &axisInfoAnalysis,
     ModuleAllocation &allocation,
     ConvertTritonGPUOpToLLVMPatternBase::IndexCacheInfo &indexCacheInfo,
-    int computeCapability, PatternBenefit benefit) {
+    int computeCapability, const TargetInfo &targetInfo,
```

Comment:
Remove cc if not needed anymore (probably not needed)

Reply:
I will actually leave this to another PR, I have to propagate the options.arch from frontend to amd backend, and I need to repalce all the cc checks in amd backend (specifically in elementwise lowering)